examples/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
+
+# Ignore IDE
+.idea
+.project
+.settings
+lib/.settings
+lib/.classpath
+lib/.project
+lib/bin
```

Comment:
Perhaps merge `examples/.gitignore` with the repo root's `.gitignore`. Otherwise the project won't pick this up. Will have to modify the paths here